### ISSUE TITLE
Frontend Refactoring

### ISSUE THREAD

**Issue by 0xduda** [sbsweb3hub/sbs_site_v.1.0#9]
Список модулей на снос:

"@emotion/react": "^11.11.1",
"@emotion/styled": "^11.11.0",
"@mui/x-charts": "^6.18.3",
"@rainbow-me/rainbowkit": "^1.3.0",  (под вопросом)
"@uniswap/sdk-core": "^4.0.9",
"@uniswap/v3-sdk": "^3.10.0",
"chart.js": "^4.4.1",
"viem": "^1.19.11", (под вопросом)
"wagmi": "^1.4.10" (под вопросом)
 "v3-periphery": "^1.2.1", (не понял что это, надо?)
"@faker-js/faker": "^8.3.1",

Теперь предложение по структуре. Логично что все компоненты должны лежать в папке Components, которая там есть, и в ней уже есть часть старых компонентов. Сейчас куча компонентов лежит в рядом с основным пэйджом + что-то старое и не нужное там точно есть. В общем предлагаю следующую структуру:

1. В папке Components храним ВСЕ компоненты.

2. В самой папке папке Components создаём папки хотя под часто повтояющиеся компоненты по смыслу. Вот у нас сейчас есть куча модалок, которые мы можем положить в папку Modals в папке Components. 

3. Тоже самое я предлагаю сделать для компонентов, которые относятся к определённой странице. Например, сейчас у нас есть компонент StartPage, который засунут в основной page. В StartPage идёт куча других компонентов. Поэтому вижу структуру следующей, в папке Components создаём папку StartPage - в StartPage кладём компонент StartPage.tsx и делаем рядом папку StartPageContent, куда кладём уже остальные компоненты которые идут в этот StartPage

**Comment by Serggoal** [comment]
На снос: 
в папке create
- Selector4.tsx
- TableAmountCreate.tsx

**Comment by 0xduda** [comment]
<img width="327" alt="Screenshot 2567-04-20 at 11 11 51" src="https://github.com/sbsweb3hub/sbs_site_v.1.0/assets/148529985/172a4115-0acb-4c3d-afc4-9c9802303ea9">

Теперь это будет выглядеть так? Пожелания?

**Comment by 0xduda** [comment]
@Serggoal вот эти нужны?
 
 
<img width="301" alt="Screenshot 2567-04-20 at 11 14 27" src="https://github.com/sbsweb3hub/sbs_site_v.1.0/assets/148529985/97adae02-6dec-404c-910a-432b7e07d590">

**Comment by Serggoal** [comment]
> @Serggoal вот эти нужны?
> 
> <img alt="Screenshot 2567-04-20 at 11 14 27" width="301" src="https://private-user-images.githubusercontent.com/148529985/324158314-97adae02-6dec-404c-910a-432b7e07d590.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM2MDI4ODMsIm5iZiI6MTcxMzYwMjU4MywicGF0aCI6Ii8xNDg1Mjk5ODUvMzI0MTU4MzE0LTk3YWRhZTAyLTZkZWMtNDA0Yy05MTBhLTQzMmI3ZTA3ZDU5MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyMFQwODQzMDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMzNlNTE1YzE4OTY3ZWM3MDlkMzA1YmZlMGE0N2FhNDZlMzkwOWMzYTliYTI0NmE0MDlhZjI4YTI5YWU1ZDMzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.g1K8mRN33e0SmVA-CG5-in3zuZd5Mt4TUwgQY_ehkbU">

нет, но ты после удаления проверяй на всякий случай работоспособность приложения)) кажется всё это нигде не используется
